src/engraving/tests/parts_data/part-chordline-add.mscx

```diff
@@ -878,6 +878,11 @@
                   </linked>
                 <pitch>74</pitch>
                 <tpc>16</tpc>
+                <ChordLine>
```

Comment:
Because the same problem that was happening when cloning from staff to TAB, was also happening when cloning from score to part (see Simon's [comment here](https://github.com/musescore/MuseScore/issues/15764#issuecomment-1438383553)). So basically the ChordLine was missing from the part in the unit test. Now the PR fixed that so I had to add it in